Lib/test/test_decimal.py

```diff
@@ -1111,6 +1111,12 @@ def test_formatting(self):
             ('x>z6.1f', '-0.', 'xxx0.0'),
             ('🖤>z6.1f', '-0.', '🖤🖤🖤0.0'),  # multi-byte fill char
```

Comment:
Is there a test for 'z' format with null fill character?
```suggestion
            ('🖤>z6.1f', '-0.', '🖤🖤🖤0.0'),  # multi-byte fill char
            ('\0>z6.1f', '-0.', '\0\0\00.0'),  # null fill char
```

Reply:
this added null fill test fails

```
AssertionError: '\x00\x00\x000.0' != '\x00\x00\x00.0'
- 0.0
?    -
+ .0
```

I'm not sure which is correct, nor how the new mpdecimal `z` implementation will respond, so perhaps this is for another PR.

Reply:
(Serhiy corrected the test-- thank you)